SECURITY.md

```diff
@@ -42,20 +42,25 @@ Certain members of the Magma team have been designated as the "vulnerability man
 
 5. Decide whether the report calls for a formal advisory in addition to an issue. 
 
-6. If it does not call for an advisory, inform the discloser, taking care not to reveal potential weaknesses, then prioritize and fix the issue according to ordinary workflows.
+  a. If it does not call for an advisory, inform the discloser, taking care not to reveal potential weaknesses, then prioritize and fix the issue according to ordinary workflows.
 
-If the report does merit an advisory, the vulnerability management team will:
+  b. If the report does merit an advisory, the vulnerability management team will execute the Advisory Procedure below
+  
+6.  Add the discloser to the security acknowledgements section in README.md. 
+
+		
+### Advisory Procedure
 
 1. Assign a priority as follows:
 
-   - Critical Risk: shall be treated as "P0" issue, with a fix developed and released as soon as possible even if it requires an out-or-sequence release.
-   - High Risk: shall be treated as "P1" issue, with a fix delivered no later than the next scheduled "intermediate" release.
-   - Medium Risk: shall be treated a "P1" issue, with a fix delivered no later than the next scheduled "intermediate" release.
-   - Low Risk: shall be treated as a "P2" issue, with a fix delivered no later than the next scheduled "major" release
+- Critical Risk: shall be treated as "P0" issue, with a fix developed and released as soon as possible even if it requires an out-or-sequence release.
+- High Risk: shall be treated as "P1" issue, with a fix delivered no later than the next scheduled "intermediate" release.
+- Medium Risk: shall be treated a "P1" issue, with a fix delivered no later than the next scheduled "intermediate" release.
+- Low Risk: shall be treated as a "P2" issue, with a fix delivered no later than the next scheduled "major" release
```

Comment:
This was correctly formatted before and should not be changed.

Reply:
Indented.